### PR TITLE
Set a reading order to the compatibility import file

### PR DESCRIPTION
**Pull request by agamrafaeli**
The order is as follows:
1. Imports, in the same format of "import X"
2. Members denoting versions of python
3. Members of imports that are shadowed for usage across the system
4. Global variables
5. If conditions of imports. (internally organized by this order)
6. Function definitions

### CODE REVIEW

**Comment by agamrafaeli** [comment]
Happy to hear other conventions and suggestions for order and then do the grunt work on my own.

**Review by sqla-tester — COMMENTED** [review]
OK, this is **sqla-tester** setting up my work to try to get this PR into gerrit so we can run tests and reviews and stuff

**Comment by sqla-tester** [comment]
Change has been squashed to Gerrit review: https://gerrit.sqlalchemy.org/#/c/sqlalchemy/sqlalchemy/+/1036

File: lib/sqlalchemy/util/compat.py [hunk 1]
```diff
@@ -327,12 +302,32 @@ def nested(*managers):
                 exc = sys.exc_info()
         if exc != (None, None, None):
             reraise(exc[0], exc[1], exc[2])
+    exc = (None, None, None)
```
Comment (zzzeek) [inline lib/sqlalchemy/util/compat.py]: this line moving from up above is what broke the tests in https://jenkins.sqlalchemy.org/job/sqlalchemy_gerrit/cext=cext,db=sqlite-postgresql,pyv=py27/1930/#showFailuresLink, can you move that back?   you can re-push to this PR and I should be able to continue from there.

**Review by sqla-tester — COMMENTED** [review]
OK, this is **sqla-tester** setting up my work to try to get this PR into gerrit so we can run tests and reviews and stuff

**Comment by sqla-tester** [comment]
Change has been squashed to Gerrit review: https://gerrit.sqlalchemy.org/#/c/sqlalchemy/sqlalchemy/+/1036

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/1036 has been **merged**. Congratulations! :)

**Pull request closed without merging**